From 506484d71bb8d8b8d68b6895364cb183fe2b24ee Mon Sep 17 00:00:00 2001
From: Pavel Grunt <pgrunt@redhat.com>
Date: Tue, 7 Feb 2017 16:31:42 +0100
Subject: [PATCH 13/26] Update for README.md

It is needed to use the 'foreign' init option otherwise autotools
requires README

Fix make distcheck and spec file generation

Acked-by: Christophe Fergeau <cfergeau@redhat.com>
---
 Makefile.am         | 1 +
 configure.ac        | 2 +-
 virt-viewer.spec.in | 2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 6c7a3f9..d33367a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -18,6 +18,7 @@ EXTRA_DIST =					\
 	build-aux/useless-if-before-free	\
 	build-aux/vc-list-files			\
 	AUTHORS.in				\
+	README.md				\
 	$(NULL)
 
 DISTCLEAN_FILES =				\
diff --git a/configure.ac b/configure.ac
index 81cb576..3d03687 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5,7 +5,7 @@ AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_HEADERS([config.h])
 dnl Make automake keep quiet about wildcards & other GNUmake-isms
-AM_INIT_AUTOMAKE([subdir-objects -Wno-portability])
+AM_INIT_AUTOMAKE([subdir-objects -Wno-portability foreign])
 AC_CANONICAL_HOST
 
 # Use the silent-rules feature when possible.
diff --git a/virt-viewer.spec.in b/virt-viewer.spec.in
index 7e972c4..44ef92e 100644
--- a/virt-viewer.spec.in
+++ b/virt-viewer.spec.in
@@ -119,7 +119,7 @@ fi
 
 %files -f %{name}.lang
 %defattr(-,root,root,-)
-%doc README COPYING AUTHORS ChangeLog NEWS
+%doc README.md COPYING AUTHORS ChangeLog NEWS
 %{_bindir}/%{name}
 %{_bindir}/remote-viewer
 %{_datadir}/icons/hicolor/*/apps/*
-- 
2.12.0

